src/bgw/job_stat.h

```diff
@@ -16,6 +16,7 @@ typedef struct BgwJobStat
 
 typedef enum JobResult
 {
+	JOB_FAILURE_TO_START = -1,
```

Comment:
why -1 instead of 2?

Reply:
Because I want to reserve positive numbers for successes

Reply:
Could you add a comment before the enum about this so that future developers also know this convention?

Reply:
comment added